plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImpl.kt

```diff
@@ -694,8 +701,9 @@ internal class CameraAnimationsPluginImpl : CameraAnimationsPlugin {
 
   override fun createBearingAnimator(
     options: CameraAnimatorOptions<Double>,
+    optimizePath: Boolean,
```

Comment:
The optimizePath naming doens't indicate how it's optimizing. Would it make sense renaming this to `shortestPath` or name similar to this?

Reply:
Indeed, using `useShortestPath` flag now.